core/controllers/topic_viewer.py

```diff
@@ -168,6 +169,10 @@ def get(self, topic_name: str) -> None:
             for skill_id in all_skill_ids:
                 degrees_of_mastery[skill_id] = None
 
+        classroom_name = (
```

Comment:
Let's cover it in a test, that a classroom is returned when it exists. It could be added to an existing test as well

Reply:
Done.